mono/eglib/eglib-config.hw

```diff
@@ -17,23 +18,16 @@
 #define G_BREAKPOINT()           __debugbreak()
 #define MAXPATHLEN 242
 
-typedef uintptr_t gsize;
-typedef intptr_t gssize;
+typedef size_t gsize;
+typedef ptrdiff_t gssize;
```

Comment:
Why not ssize_t?

Reply:
That is much newer. Same thing, nicer name, granted.

Reply:
Please switch to ssize_t granted it passes CI.

Reply:
type 1.c

#include <stddef.h>

cl -EP 1.c | findstr  size_t
Microsoft (R) C/C++ Optimizing Compiler Version 19.12.25835 for x86

1.c
    typedef unsigned int     size_t;
    typedef size_t rsize_t;

cl -EP 1.c | findstr  ptrdiff
Microsoft (R) C/C++ Optimizing Compiler Version 19.12.25835 for x86

1.c
    typedef int              ptrdiff_t;

Reply:
So I suggest ptrdiff_t is really preferred for portability.
I tried stdint.h too and it isn't there.

Reply:
Makes sense to use ptrdiff_t then.